packages/ui/src/1_atoms/ErrorBadge/ErrorBadge.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render } from '@testing-library/react';
+
+import React from 'react';
+
+import { ErrorBadge } from './ErrorBadge';
+import { ErrorLevel } from './ErrorBadge.types';
+
+describe('ErrorBadge', () => {
+  it('renders a ErrorBadge', () => {
+    const { getByTestId } = render(
+      <ErrorBadge
+        message="recovery mode"
+        level={ErrorLevel.Warning}
+        dataAttribute="error"
+      />,
+    );
+    const errorBadge = getByTestId('error');
+    expect(errorBadge).toBeInTheDocument();
+  });
```

Comment:
```suggestion
```

This only tests sub-set of functionality from 2nd test, consider adjusting to test level icons or another attribute not covered by other tests.